docs/data-sources/stacks.md

```diff
@@ -0,0 +1,40 @@
+# Stack Data Source
+
+[Stacks](https://deploy-docs.aptible.com/docs/stacks)
+are the underlying virtualized infrastructure (EC2 instances, private network, etc.) your resources
+are deployed on.
+
+## Example Usage
+
+### Determining the Stack ID
+
+If you have an Stack with the name "test-stack" you can
+create the data source:
+
+```hcl
+data "aptible_stack" "test-stack" {
+    name = "test-stack"
+}
+```
+
+Once defined, you can use this data source in your resource definitions.
+For example, when defining an environment:
+
+```hcl
+resource "aptible_environment" "test-env" {
+    env_id = data.aptible_stack.test-stack.stack_id
+    name = "test-env"
+}
+```
+
+## Argument Reference
+
+- `name` - The name of the Stack
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+- `id` - The unique ID for an Stack suitable for use in `stack_id` attributes
```

Comment:
Should be `stack_id`. I assume this was copied from the environment data source doc which itself should be `env_id`, not `id`. Technically we set `id` to the stack name.

```suggestion
- `stack_id` - The unique ID for an Stack suitable for use in `stack_id` attributes
```

Reply:
Could you please update the environment data source doc as well?

Reply:
I need to update environment data source